tell/execute_forward.py

```diff
@@ -253,7 +253,7 @@ def output_tell_county_data(joint_mlp_df, data_output_dir, year_to_process):
         state_name = state_name.replace(" ", "_")
         state_name = state_name.replace(",", "_")
         csv_output_filename = os.path.join(
-            data_output_dir + '\\County_Level_Data\\TELL_' + state_name + '_' + county_name + '_Hourly_Load_Data_' + year_to_process + '.csv')
+            data_output_dir + '/County_Level_Data/TELL_' + state_name + '_' + county_name + '_Hourly_Load_Data_' + year_to_process + '.csv')
```

Comment:
backslash is specific to windows, but Python > 3.something correctly interprets forward slashes regardless of operating system

Reply:
I see, thanks for resolving! This is better for the non windows users